Documentation/operations/upgrade.rst

```diff
@@ -320,6 +320,10 @@ Removed Options
 
 * The ineffective ``disable-conntrack``, ``endpoint-interface-name-prefix`` options deprecated in
   version 1.12 have been removed.
+* The ``host-reachable-services-protos`` option deprecated in version v1.12 has
+  been removed.
+* The ``probe`` option of ``kube-proxy-replacement`` deprecated in version v1.12
+  has been removed.
```

Comment:
What is the recommendation we provide to folks who have been running this before? We should probably provide a short blurb in here.

Reply:
Aye, good idea. Will add some.